introduction_to_amazon_algorithms/jumpstart-foundation-models/text2text-generation-flan-t5-Batch-Transform.ipynb

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
<p>explain that this will be passed as an env variable to the Batch transform job. And, if any of these are passed, then the advanced hyper-parameters from the individual examples in the JSONLINES payload will not be used. If you don't want to pass it, set it to None, and in that case advanced hyper-parameters from the individual examples in the JSONLINES payload will be used and inference will be run on each example at a time.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Sure</p>